procsim/biomass/level0_product_generator.py

```diff
@@ -136,15 +136,29 @@ def _is_partial_slice(self, valid_start, valid_end, acq_start, acq_end):
         '''
         Compare validatity and acquisition times
         '''
-        start_is_aligned = valid_start == acq_start
-        end_is_aligned = valid_end == acq_end
-        if not start_is_aligned and not end_is_aligned:
-            self._logger.debug('Incomplete slice, start and end unaligned')
-        elif not start_is_aligned:
-            self._logger.debug('Incomplete slice, start unaligned')
-        elif not end_is_aligned:
-            self._logger.debug('Incomplete slice, end unaligned')
-        return not start_is_aligned or not end_is_aligned
+        delayed_start = valid_start < acq_start
```

Comment:
Meh, Github does not allow comment on non-changed lines. Anyway, the comment in line 61 should be updated as well... it now updates the partial- and merged-flags.